src/identity.cc

```diff
@@ -422,9 +482,20 @@ TRITONBACKEND_Initialize(TRITONBACKEND_Backend* backend)
       (std::string("backend configuration:\n") + buffer).c_str());
 
   // If we have any global backend state we create and set it here. We
-  // don't need anything for this backend but for demonstration
-  // purposes we just create something...
-  std::string* state = new std::string("backend state");
+  // make use of the global backend state here to track a custom metric across
+  // all models using this backend if metrics are enabled.
+  IdentityBackendState* state = new IdentityBackendState();
+
+#ifdef TRITON_ENABLE_METRICS
+  // Create metric family
+  const char* family_name = "input_byte_size_counter";
+  const char* desc =
+      "Cumulative input_byte_size across all identity model requests";
```

Comment:
the model name and version is a part of the label right?
How about for description we can leave it little more general?

`Cumulative input byte size of all requests received by the model`